pkg/pillar/cmd/client/client.go

```diff
@@ -107,8 +107,12 @@ func Run() { //nolint:gocyclo
 	}
 	defer logf.Close()
 	if useStdout {
-		multi := io.MultiWriter(logf, os.Stdout)
-		log.SetOutput(multi)
+		if logf == nil {
+			log.SetOutput(os.Stdout)
+		} else {
+			multi := io.MultiWriter(logf, os.Stdout)
+			log.SetOutput(multi)
```

Comment:
So when client logs it will alway log to a syslog and to a stdout? Btw, where does that stdout ends up these days?

Reply:
If device-steps.sh runs them, then stdout ends up in /var/log/device-steps.log.
If you run it by hand it is just stdout.
Note that diag is started by device-steps.sh with stdout and stderr redirected ton /dev/console